mappings/net/minecraft/screen/ScreenHandlerContext.mapping

```diff
@@ -8,9 +8,24 @@ CLASS net/minecraft/class_3914 net/minecraft/screen/ScreenHandlerContext
 		ARG 0 world
 		ARG 1 pos
 	METHOD method_17393 run (Ljava/util/function/BiConsumer;)V
+		COMMENT Runs a {@link BiConsumer} with this context's world and position
+		COMMENT if this context is active.
 		ARG 1 function
-	METHOD method_17395 run (Ljava/util/function/BiFunction;)Ljava/util/Optional;
-		ARG 1 function
-	METHOD method_17396 run (Ljava/util/function/BiFunction;Ljava/lang/Object;)Ljava/lang/Object;
-		ARG 1 function
+	METHOD method_17395 get (Ljava/util/function/BiFunction;)Ljava/util/Optional;
+		COMMENT Gets an optional value from this context's world and position
+		COMMENT with a {@link BiFunction} getter.
+		COMMENT
+		COMMENT @return a present {@link Optional} with the getter's return value,
+		COMMENT         or {@link Optional#empty()} if this context is empty
+		ARG 1 getter
+			COMMENT a function that gets a non-null value from this context's world and position
+	METHOD method_17396 get (Ljava/util/function/BiFunction;Ljava/lang/Object;)Ljava/lang/Object;
+		COMMENT Gets a value from this context's world and position
+		COMMENT with a {@link BiFunction} getter.
+		COMMENT
+		COMMENT @return the getter's return value if this context is active,
+		COMMENT         the default value otherwise
+		ARG 1 getter
+			COMMENT a function that gets a non-null value from this context's world and position
 		ARG 2 defaultValue
```

Comment:
I'm inclined to name all these default values `fallback`

Reply:
Hmm, that might be worth a note in `CONVENTIONS.md` since it's a relatively common pattern in the code base.

Reply:
Yeah, blame java for reserving `default`. `defaultValue` is too long compared to `fallback`.